taskcluster/ci/toolchain/android.yml

```diff
@@ -7,6 +7,19 @@ job-defaults:
     worker:
         max-run-time: 1800
 
+linux64-android-gradle-repack:
+    description: "Android SDK (Linux) repack toolchain build"
+    run:
+        script: repack-gradle.sh
+        resources: []
+        toolchain-artifact: public/build/gradle.tar.xz
+        toolchain-alias: android-gradle
+    treeherder:
+        symbol: TL(gradle)
+    worker:
+        docker-image: {in-tree: base}
+        env: {}
```

Comment:
I needed to set this key because taskgraph expects `env` to be set here https://hg.mozilla.org/ci/taskgraph/file/cab4565345d0d0effa66f18fe91335dd6d744031/src/taskgraph/transforms/job/common.py#l89. The fix should be on taskgraph, I'll do it later.